crates/api_models/src/payouts.rs

```diff
@@ -261,6 +262,28 @@ pub struct SepaBankTransfer {
     pub bic: Option<Secret<String>>,
 }
 
+#[derive(Default, Eq, PartialEq, Clone, Debug, Deserialize, Serialize, ToSchema)]
+pub struct PixBankTransfer {
+    /// Bank name
+    #[schema(value_type = Option<String>, example = "Deutsche Bank")]
+    pub bank_name: Option<String>,
+
+    /// Bank branch
+    #[schema(value_type = Option<String>, example = "3707")]
+    pub bank_branch: Option<String>,
+
+    /// Bank account number is an unique identifier assigned by a bank to a customer.
+    #[schema(value_type = String, example = "000123456")]
+    pub bank_account_number: Secret<String>,
+
+    /// Unique key for pix customer
+    #[schema(value_type = String, example = "000123456")]
+    pub pix_key: Secret<String>,
+
+    /// Individual taxpayer identification number
+    pub tax_id: Secret<String>,
```

Comment:
Will this always exist regardless of the country? Or this is LATAM Specific, please check if this is required to be made optional